.travis.yml

```diff
@@ -21,6 +21,7 @@ branches:
     - rnmobile/master
     - /rnmobile\/release.*/
     - /wp\/.*/
+    - feat/import-gutenberg-mobile-no-squash
```

Comment:
I think this should be removed before/after merge, right?

Reply:
Yeap, we're not going to hold on the branch. 👍

Reply:
Updated 👍